ImageKitSample/Program.cs

```diff
@@ -0,0 +1,385 @@
+﻿using Imagekit.Models;
+using Imagekit.Sdk;
+using System;
+using System.Collections.Generic;
+using Imagekit;
+using static Imagekit.Models.CustomMetaDataFieldSchemaObject;
+using Newtonsoft.Json.Linq;
+
+namespace ImagekitSample
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Create Instance of ImageKit
+            ImagekitClient imagekit = new ImagekitClient("TestPublicKey", "TestPrivateKey", "https://api.imagekit.io/");
```

Comment:
Please follow the convention of using 
"your_private_key"
"your_public_key"
"http://ik.imagekit.io/imagekit_id/endpoint"